pkg/p2p/libp2p/libp2p.go

```diff
@@ -285,6 +285,10 @@ func (s *Service) AddProtocol(p p2p.ProtocolSpec) (err error) {
 			// exchange headers
 			if err := handleHeaders(ss.Headler, stream); err != nil {
 				s.logger.Debugf("handle protocol %s/%s: stream %s: peer %s: handle headers: %v", p.Name, p.Version, ss.Name, overlay, err)
+				s.logger.Errorf("handle protocol %s/%s: peer %s", p.Name, p.Version, overlay)
```

Comment:
sure we want this on `error`? maybe `debug`?

Reply:
I generally agree, but there will be no message for the user then, cuz we don't return an error here. Maybe warning?

Reply:
this is not really a user-facing error anyway... not sure what the mere mortal is supposed to make of this sort of error. it is pretty obscure. i'd even go as far as saying just have the debug line above and remove this line. anyway, as you wish

Reply:
Cool, good for me :)